src/Workspaces/CSharp/Portable/Classification/Worker.cs

```diff
@@ -223,5 +219,30 @@ private void ClassifySkippedTokens(SkippedTokensTriviaSyntax skippedTokens)
                 ClassifyToken(tk);
             }
         }
+
+        private void ClassifyConflictMarker(SyntaxTrivia trivia)
+        {
+            AddClassification(trivia, ClassificationTypeNames.Comment);
```

Comment:
Should we introduce a new color here? I might want to make my conflict markers bright red. :smile: